cmd/lotus/daemon.go

```diff
@@ -329,7 +329,12 @@ var DaemonCmd = &cli.Command{
 			liteModeDeps,
 
 			node.ApplyIf(
-				func(s *node.Settings) bool { return build.IsMirConsensus() },
+				func(s *node.Settings) bool {
```

Comment:
We should leave lotus as the binary used to initialize a Filecoin client and leave Eudico as the binary used to initialize other consensus algorithms. We can remove these few lines and the ability to initialize Lotus with Mir.